### DISCUSSION TITLE
Why does the nmap package depend on autohotkey?

### DISCUSSION THREAD

**Discussion by gbakeman** [chocolatey-community/chocolatey-packages#2420] [Q&A] [answered]
Can anyone help me understand why the nmap package depends on autohotkey? I don't recall ever needing AHK before when installing nmap.

**Comment by pauby** [comment]
This exact question, and answer is in the Disqus comments, on the package page.

**Reply by gbakeman** [reply, marked as answer]
I only saw people questioning using the .installer package over the regular package, but no one questioning AHK in general. I did happen upon one comment mentioning issues with a silent install, which linked to [this issue](https://github.com/chocolatey-community/chocolatey-packages/issues/1557) specifying a workaround of using AHK so I guess that answers my question.

**Reply by pauby** [reply]
This is a questions that's been raised a few times. I think it's something we should make clearer.

Thank you for finding it. Apologies it wasn't quite as straightforward as I was sure it was (I was confident I'd answered this in Disqus in the past).
